health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/pages/employee/ViewDashboard.js

```
@@ -0,0 +1,194 @@
+      codes: project?.address?.boundary,
+      hierarchyType: hierarchyType,
+    },
+    config: {
+      enabled: !!hierarchyType,
+      select: (data) => {
+        return data?.["TenantBoundary"]?.[0]?.boundary;
+      },
+    },
+  };
+
+  const { isLoading: dataLoading, data: data, isFetching, refetch2 } = Digit.Hooks.useCustomAPIHook(reqCriteriaResource);
```

Biome (1.9.4)

[error] Useless rename.

Safe fix: Remove the renaming.


(lint/complexity/noUselessRename)